build/linux/installer/conf/fluent-bit-common.conf

```diff
@@ -77,13 +91,20 @@
     Match oms.container.log.flbplugin.mdsd.qos.*
     regex log ^MaODSRequest,https.*_LogManagement_LINUX_SYSLOGS_BLOB
 
+[FILTER]
+    Name grep
+    Alias ama-logs_amaca_log_tail
+    Match oms.container.log.flbplugin.amaca.*
+    Exclude log Information
+
 [OUTPUT]
     Name                            oms
     Alias                           oms_output
     EnableTelemetry                 true
     Retry_Limit                     10
     TelemetryPushIntervalSeconds    300
     Match                           oms.container.*
+    ${STORAGE_TOTAL_LIMIT_SIZE_MB}
```

Comment:
do we want to change the new added config to this format in fluent-bit conf? #${STORAGE_TOTAL_LIMIT_SIZE_MB} storage.total_limit_size 1000M